layouts/shortcodes/release-binaries.html

```diff
@@ -0,0 +1,130 @@
+<!-- Fetch release_binaries.json from kubernetes-sigs/downloadkubernetes to render in table -->
+{{ $response := getJSON "https://raw.githubusercontent.com/kubernetes-sigs/downloadkubernetes/master/dist/release_binaries.json" }}
+
+{{ $currentVersion := site.Params.version }}
+
+{{ $Binaries := slice }}
+{{ $AllOSes := slice }}
+{{ $AllArch := slice }}
+{{ $AllVersions := slice }}
+
+{{ range $key, $value := $response }}
+    {{ if eq $key "Binaries" }}
+        {{ $Binaries = $value }}
+    {{ else if eq $key "AllOSes" }}
+        {{ $AllOSes = $value }}
+    {{ else if eq $key "AllArch" }}
+        {{ $AllArch = $value }}
+    {{ else if eq $key "AllVersions" }}
+        {{ $AllVersions = $value }}
+    {{ end }}
+{{ end }}
+
+
+<!-- The below <div> defines an alternate content to be displayed only 
+     when Javascript is disabled or when user's browser doesn't support Javascript -->
+<div class="downloadbinaries-nojs">
+        <p>{{ T "release_binary_alternate_links" | markdownify }}</p>
+</div>
+
+<!-- The below section containing the release binary details
+    is enabled by the script "release_binaries.js"  -->
+<div id="download-kubernetes-data" style="display: none;">
+    <p>
+        {{ $releaseBinarySection := T "release_binary_section" }}
+        {{ $releaseBinarySection = replace $releaseBinarySection "<current-version>" $currentVersion }}
+        {{ $releaseBinarySection | markdownify }}
+    </p>
+    <div class="alert alert-info note callout" role="alert">
+        <strong>{{ T "note" }}</strong>
+        {{ $releaseBinarySectionNote := T "release_binary_section_note" }}
+        {{ $releaseBinarySectionNote = replace $releaseBinarySectionNote "<current-changelog-url>" (replace $currentVersion "v" "") }}    
+        {{ $releaseBinarySectionNote = replace $releaseBinarySectionNote "<current-version>" $currentVersion }}
+        {{ $releaseBinarySectionNote | markdownify }}
+    </div>    
```

Comment:
1. Ideally, this is conditional: if the patch version is the first patch for a release, we omit it.
    I'm happy to see this merge and for that to come in as a follow up.
1. Ideally, this would call the `note` shortcode rather than copy and paste it in.